docs/guide/controller-programming.md

```diff
@@ -98,11 +233,47 @@ The call to the `wb_distance_sensor_get_value` function retrieves the latest val
 
 Note that some device return vector values instead of scalar values, for example these functions:
 
+%tab-component "language"
+%tab "C"
 ```c
 const double *wb_gps_get_values(WbDeviceTag tag);
 const double *wb_accelerometer_get_values(WbDeviceTag tag);
 const double *wb_gyro_get_values(WbDeviceTag tag);
 ```
+%tab-end
+
+%tab "C++"
+```cpp
+webots::GPS::const double *getValues() const;
+webots::Accelerometer::const double *getValues() const;
+webots::Gyro::const double *getValues() const;
+```
+%tab-end
+
+%tab "Python"
+```python
+[x, y, z] GPS.getValues()
+[x, y, z] Accelerometer.getValues()
+[z, y, z] Gyro.getValues()
+```
```

Comment:
Here is what I mean:

```python
GPS.getValues()
Accelerometer.getValues()
Gyro.getValues()
```
Returns the sensor measurement as an array of 3 floating point numbers: `[x, y, z]`.